src/app/containers/LoginPage/validators.js

```diff
@@ -0,0 +1,28 @@
+const MSG_ERROR_FIELD_IS_REQUIRED = 'Mục này là bắt buộc';
+const MSG_ERROR_INVALID_EMAIL = 'Email không hợp lệ';
```

Comment:
rename to `MSG_ERROR_EMAIL_INVALID` so that the order is consistent: 1. subject, 2. description